lambdas/retrieve-cri-credential/src/test/java/uk/gov/di/ipv/core/retrievecricredential/RetrieveCriCredentialHandlerTest.java

```diff
@@ -98,6 +112,8 @@ class RetrieveCriCredentialHandlerTest {
     private static CriOAuthSessionItem criOAuthSessionItem;
     @InjectMocks private RetrieveCriCredentialHandler handler;
 
+    @Captor private ArgumentCaptor<List<String>> postedVcsCaptor;
```

Comment:
this captor is using only one method.

Reply:
It seemed to be the only clean way to create an ArgumentCaptor<List<String>> but I had another crack at it. Sorted